test/unit/mailers/account_mailer_test.rb

```diff
@@ -74,7 +74,7 @@ def test_support_entitlements_assigned
 
     def test_support_entitlements_trial_plan
       trial_plan = FactoryBot.create(:application_plan, issuer: @service)
-      trial_plan.plan_rule.metadata = {trial: true}
+      trial_plan.send(:plan_rule).metadata = {trial: true}
```

Comment:
why this change? Is it not equivalent?

Reply:
mocha 2.x stubs private methods as private. Previously all was stubbed public. So we can't call directly anymore.

Reply:
I wonder why ruby implements private methods if private protection is so easy to bypass.

Reply:
It's to notify the user more than enforce restrictions. Still somehow useful.